### DISCUSSION TITLE
where is `accountability` from?

### DISCUSSION THREAD

**Discussion by anlexN** [directus/directus#15513] [Archive]
![image](https://user-images.githubusercontent.com/8686739/189482346-4cced6bb-a48e-43bd-979d-7789ab575718.png)
**`accountability` shows up in a lot of places in the source code.**
I wonder `accountability` 
what book is this word from? what theory? what protocol? or say what standard?

**Comment by rijkvanzanten** [comment]
> what book is this word from?

.. the dictionary?

> what theory?

.. mine? 

> what protocol?

.. none?

> or say what standard?

No idea, we designed this ourselves! 

---

On a more serious note, "accountability" is about who's accountable/responsible for a change, so it felt appropriate as a name for the thing that describes the entity who's operating against the service

**Reply by anlexN** [reply]
I recommend to find word in [schema.org](https://schema.org/)
